src/FixerConfiguration/FixerConfigurationResolver.php

```diff
@@ -67,15 +67,9 @@ public function resolve(array $options)
             $allowedValues = $option->getAllowedValues();
             if (null !== $allowedValues) {
                 foreach ($allowedValues as &$allowedValue) {
-                    if ($allowedValue instanceof AllowedValueSubset) {
+                    if (is_object($allowedValue) && is_callable($allowedValue)) {
```

Comment:
is `is_object` check really needed ?

Reply:
Yes, as an array like e.g. `[$object, 'method']` would be callable as well.